src/components/ArticleV3/SimpleArticleCell.js

```diff
@@ -25,24 +25,45 @@ const ArticleCell = ({
   isJavascriptTrusted = false,
   onNumClick,
   thebeCell,
-  executing,
-  executeCell,
+  useThebeRendering,
+  notebookIsExecuting,
```

Comment:
this mnotebookIsExecuting flag will disappear once we have proper state mangement in place for triggering execution, busy & errors